src/text/text.rs

```diff
@@ -240,6 +334,42 @@ impl std::fmt::Display for Text<'_> {
     }
 }
 
+impl<'a> Widget for Text<'a> {
+    fn render(self, area: Rect, buf: &mut Buffer) {
+        buf.set_style(area, self.style);
+        for (line, row) in self.lines.into_iter().zip(area.rows()) {
+            let line_width = line.width() as u16;
+
+            let x_offset = match (self.alignment, line.alignment) {
+                (Some(Alignment::Center), None) => (area.width / 2).saturating_sub(line_width / 2),
```

Comment:
I think (could be wrong) that this might have some odd behavior with the way it rounds down twice
area: 5, line: 3 = 2-1 = 1 "xYYYx"
area: 6, line 3 = 3-1 = 2 "xxYYYx"
area: 5, line 4 = 2-2 = 0 "YYYYx"
area: 6, line 4 = 3-2 = 1 "xYYYYx"

If you do the subtraction first:
(5-3)/2 = 1
(6-3)/2 = 1 "xYYYxx"
(5-4)/2 = 0
(6-4)/2 = 1

I think having centered end up a little to the left is a bit more consistent (with the 3rd and 4th line)

Reply:
You're right, changed it and added a test for that.